### PR TITLE
fixes "CSV export is limited to 10000 events" menu is hidden behind export menu

### PR DESCRIPTION
**Pull request by ahsan722505**
…ort menu

## Problem

"CSV export is limited to 10000 events" menu is hidden behind export menu. #17810 

## Changes
Screenshot from 2023-10-06 11-21-24

### CODE REVIEW

File: frontend/src/queries/nodes/DataTable/ExportWithConfirmation.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ export function ExportWithConfirmation({
 }: ExportWithConfirmationProps): JSX.Element {
     return (
         <Popconfirm
+            zIndex={10000}
```
Comment (mariusandra) [inline frontend/src/queries/nodes/DataTable/ExportWithConfirmation.tsx:21]: This is likely going to mess up the ordering of other things.
We prefer to use z-indexes as a CSS variables and have a list of them [here](https://github.com/PostHog/posthog/blob/dfd76f4c71103e079992206df3e62c87c6b9bdcf/frontend/src/styles/vars.scss#L251-L274).

**Comment by ahsan722505** [comment]
comment is addressed!

**Comment by neilkakkar** [comment]
also fixes https://github.com/PostHog/posthog/issues/17601

**Review by mariusandra — APPROVED** [review]
Looks good, thanks for this!

**Pull request merged**